mapper/tedge_mapper/src/sm_c8y_mapper/mapper.rs

```diff
@@ -202,4 +219,192 @@ impl CumulocitySoftwareManagement {
             .await?;
         Ok(())
     }
+
+    async fn send_software_list_http(
+        &self,
+        json_response: &SoftwareListResponse,
+    ) -> Result<(), SMCumulocityMapperError> {
+        let token = get_jwt_token(&self.client).await?;
```

Comment:
It is, but the token has relatively short life (by default 1h) and as we don't know when it is going to be used it may be a hit and miss, and it makes sense to retrieve on each occasion.